src/features/comments.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-param-reassign */
+
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { Comment } from '../types/Comment';
+import { createComment, deleteComment, getPostComments } from '../api/comments';
+
+export const getComments = createAsyncThunk(
+  'comments/fetch',
+  (postId: number) => getPostComments(postId),
+);
+
+export const createNewComment = createAsyncThunk(
+  'comments/create',
+  (data: Omit<Comment, 'id'>) => createComment(data),
+);
+
+export const deleteAPIComment = createAsyncThunk(
+  'comments/delete',
+  (commentId: number) => deleteComment(commentId),
+);
+
+type Users = {
+  comments: Comment[];
+  loaded: boolean;
+  hasError: boolean;
+};
+const initialState: Users = {
+  comments: [],
+  loaded: true,
```

Comment:
Consider setting `loaded` to `false` in the initial state to reflect that comments are not loaded initially. This aligns better with typical loading state management.